### DISCUSSION TITLE
HW3 part 1

### DISCUSSION THREAD

**Discussion by AmanuelDagnew** [cooperrc/computational-mechanics#163] [Q&A]
How do I load the npz file-projectile_coords.npz? 
I tried this :- 
npz_coords = np.load('projectile_coords.npz')
print(npz_coords)
I get errno2.

**Comment by C-Doll** [comment]
Try using this line of code to load the data 
npz_coords = np.load('../data/projectile_coords.npz')

The code need to know where to look and sometimes it may need more information.

**Reply by AmanuelDagnew** [reply]
It worked! Thank you so much.
